Documentation/storyboards/osx-swift2.md

```diff
@@ -2,8 +2,8 @@
 
 | Name      | Description       |
 | --------- | ----------------- |
-| File name | storyboards-osx-default.stencil |
-| Invocation example | `swiftgen storyboards -t osx-default …` |
+| File name | storyboards/macOS-swift2.stencil |
```

Comment:
Those documentation files themselves should be renamed `macOS-swift2.md`, etc. right?